.gitignore

```diff
@@ -41,7 +41,26 @@ myvalues.yml
 
 vendor/**
 hack/code-generator/**
+hack/code-generator/**
```

Comment:
seems to be twice

Reply:
Yeah, actually this should be removed as `hack` no longer exists